src/chttpd/src/chttpd_db.erl

```diff
@@ -505,148 +570,179 @@ db_req(#httpd{method='POST', path_parts=[_, <<"_bulk_get">>],
+        Keys when is_list(Keys) ->
+            all_docs_view(Req, Db, Keys, OP);
+        nil ->
+            all_docs_view(Req, Db, undefined, OP);
+        _ ->
+            throw({bad_request, "`keys` body member must be an array."})
     end;
-
-db_req(#httpd{path_parts=[_,OP]}=Req, _Db) when ?IS_ALL_DOCS(OP) ->
+db_req(#httpd{path_parts = [_, OP]} = Req, _Db) when ?IS_ALL_DOCS(OP) ->
     send_method_not_allowed(Req, "GET,HEAD,POST");
-
-db_req(#httpd{method='POST',path_parts=[_,<<"_missing_revs">>]}=Req, Db) ->
+db_req(#httpd{method = 'POST', path_parts = [_, <<"_missing_revs">>]} = Req, Db) ->
     chttpd:validate_ctype(Req, "application/json"),
     {JsonDocIdRevs} = chttpd:json_body_obj(Req),
     case fabric2_db:get_missing_revs(Db, JsonDocIdRevs) of
         {error, Reason} ->
             chttpd:send_error(Req, Reason);
         {ok, Results} ->
-            Results2 = [{Id, couch_doc:revs_to_strs(Revs)} ||
-                    {Id, Revs, _} <- Results],
-            send_json(Req, {[
-                {missing_revs, {Results2}}
-            ]})
+            Results2 = [
+                {Id, couch_doc:revs_to_strs(Revs)}
+             || {Id, Revs, _} <- Results
+            ],
+            send_json(
+                Req,
+                {[
+                    {missing_revs, {Results2}}
+                ]}
+            )
     end;
-
-db_req(#httpd{path_parts=[_,<<"_missing_revs">>]}=Req, _Db) ->
+db_req(#httpd{path_parts = [_, <<"_missing_revs">>]} = Req, _Db) ->
     send_method_not_allowed(Req, "POST");
-
-db_req(#httpd{method='POST',path_parts=[_,<<"_revs_diff">>]}=Req, Db) ->
+db_req(#httpd{method = 'POST', path_parts = [_, <<"_revs_diff">>]} = Req, Db) ->
     chttpd:validate_ctype(Req, "application/json"),
     {JsonDocIdRevs} = chttpd:json_body_obj(Req),
     case fabric2_db:get_missing_revs(Db, JsonDocIdRevs) of
         {error, Reason} ->
             chttpd:send_error(Req, Reason);
         {ok, Results} ->
             Results2 =
-            lists:map(fun({Id, MissingRevs, PossibleAncestors}) ->
-                {Id,
-                    {[{missing, couch_doc:revs_to_strs(MissingRevs)}] ++
-                        if PossibleAncestors == [] ->
-                            [];
-                        true ->
-                            [{possible_ancestors,
-                                couch_doc:revs_to_strs(PossibleAncestors)}]
-                        end}}
-            end, Results),
+                lists:map(
+                    fun({Id, MissingRevs, PossibleAncestors}) ->
+                        {Id, {
+                            [{missing, couch_doc:revs_to_strs(MissingRevs)}] ++
+                                if
+                                    PossibleAncestors == [] ->
+                                        [];
+                                    true ->
+                                        [
+                                            {possible_ancestors,
```

Comment:
0. I would prefer
```
{
    possible_ancestors,
    couch_doc:revs_to_strs(PossibleAncestors)
}
```

Reply:
If formatted like that manually (or just if there's a newline between `{` and first element, erlfmt will produce the format you proposed